### ISSUE TITLE
A few issues with randomizing minions

### ISSUE THREAD

**Issue by Krisstal** [lushen124/Universal-FE-Randomizer#22]
In FE 8 at least, if you choose to randomize generic enemies, generic healers like the ones near Orson in chapter 16 keep their staves and don't get any new weapons when randomized into another class.

If a generic enemy gets turned into a Manakete, they lose everything in their inventory, when they should have at least a Dragonstone.

And lastly, in chapter 19, all enemies are still the same, including Riev, who is still a Bishop with a droppable Aura when he appeared as a Summoner in every other chapter.

Edit: I'd also like to say how incredibly hard the early game of FE7 HHM is if you choose to randomize generic enemies, since most brigands get randomized into wyvern riders or pegasus knights and they can just fly over rivers and kill pretty much any sword user, especially in chapter 13x, which is filled to the brim with enemy fliers.

**Comment by lushen124** [comment]
Thanks. I'll take a look at those chapters. I suspect the parsing logic for those chapters is bugged and it's not seeing the chapter units. Just for reference, are you on Eirika or Ephraim mode? Those chapters are technically different.

As for Manaketes, I'm going to disable Manaketes from the pool since they don't seem particularly balanced for randomization and tend to cause issues. I may reintroduce them at some later point once I figure out how I want to deal with them.

I'll probably modify the logic to limit class selection for random minions to stick to a flying/non-flying class based on their normal class so that you don't get overrun with fliers all over the place. Alternatively, I may just decrease the chance of flying classes in general from enemies.

**Comment by Krisstal** [comment]
Yeah, chapter 19 is still the same both in Eirika's and Ephraim's route.

I dunno, the first Manakete I encountered was in Chapter 15. I gave them a Dragonstone manually and they weren't too hard to deal with, since by that point of the game I had guys capable of taking them down. And they're still weak to arrows. As long as they can only replace promoted enemies they should be fine.

And thanks, I'm literally unable to beat HHM Chapter 13x without any of my guys dying in the first turn.
